nuxeo-webdav-test/src/test/java/org/nuxeo/ecm/webdav/WebDavClientTest.java

```diff
@@ -202,6 +202,33 @@ public void testListFolderContents() throws Exception {
         }
     }
 
+    @Test
+    public void testListFolderContentsSpecialName() throws Exception {
+        String folderName = "foo bar[1] caf\u00e9";
+        DocumentModel folder = session.createDocumentModel("/workspaces/workspace", folderName, "Folder");
+        session.createDocument(folder);
+        session.save();
+        TransactionHelper.commitOrRollbackTransaction();
+        TransactionHelper.startTransaction();
```

Comment:
Use `Transactionalfeature#nextTransaction()` instead?

Reply:
And same remarks for the other tests.

Reply:
The pre-existing tests use both idioms and I copy/pasted one... This will need to be cleaned up later.